README.md

```diff
@@ -493,8 +489,6 @@ Services to securely store your Docker images.
 * [Container solutions](http://container-solutions.com/blog/)
 * [Container42](http://container42.com/)
 * [Docker Cheat Sheet](http://docker.jens-piegsa.com) by [@JensPiegsa][JensPiegsa] *(updated for Docker 1.13)*
-* [Docker Cheat Sheet][docker-cheat-sheet] by [@wsargent][wsargent] __MUST SEE__
```

Comment:
The only comment I would offer is to leave the docker cheat sheet in both places because it is one of the high quality and tightly maintained project. What do you think about that ? 
The reason for this is that I still use it sometimes but it is also a great resource for beginners.